interface/billing/sl_eob_search.php

```diff
@@ -550,616 +554,699 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
-                        <div class="col-sm-12 position-override oe-show-hide" id="search-btn">
-                            <div class="btn-group" role="group">
-                                <button type='submit' class="btn btn-default btn-search oe-show-hide" name='form_search'
-                                id="btn-inv-search" value='<?php echo xla("Search"); ?>'><?php echo xlt("Search"); ?></button>
-                                <button type='submit' class="btn btn-default btn-save oe-show-hide" name='form_search'
-                                id="btn-era-upld" value='<?php echo xla("Upload"); ?>'><?php echo xlt("Upload"); ?></button>
-                            </div>
+                    </div>
+                </fieldset>
+                <?php //can change position of buttons by creating a class 'position-override' and adding rule text-alig:center or right as the case may be in individual stylesheets ?>
+                <div class="form-group clearfix">
+                    <div class="col-sm-12 position-override oe-show-hide" id="search-btn">
+                        <div class="btn-group" role="group">
+                            <button type='submit' class="btn btn-default btn-search oe-show-hide" name='form_search'
+                                    id="btn-inv-search"
+                                    value='<?php echo xla("Search"); ?>'><?php echo xlt("Search"); ?></button>
+                            <button type='submit' class="btn btn-default btn-save oe-show-hide" name='form_search'
+                                    id="btn-era-upld"
+                                    value='<?php echo xla("Upload"); ?>'><?php echo xlt("Upload"); ?></button>
                         </div>
                     </div>
-                    <fieldset id="search-results" class= "oe-show-hide">
-                        <legend><?php echo xlt('Search Results');?></legend>
-                        <div class = "table-responsive">
-                            <?php
-                            if ($_POST['form_search'] || $_POST['form_print']) {
-                                if (!verifyCsrfToken($_POST["csrf_token_form"])) {
-                                    die(xlt('Authentication Error'));
+                </div>
+                <fieldset id="search-results" class="oe-show-hide">
+                    <legend><span><?php echo xlt('Search Results'); ?></span>
+                        <div class="pull-right oe-legend-radio">
+                            <label class="checkbox-inline">
+                                <input type="checkbox" id="posting_adj_disable" name="posting_adj_disable"
+                                       onchange='persistCriteria(this, event)'
+                                       title="<?php echo xlt("Disable automatically calculating balance adjustments for invoice posting") ?>"
+                                       value="<?php echo $posting_adj_disable; ?>"
```

Comment:
yes, I finally found something :)
needs an attr() here

Reply:
I don't know why I have such an aversion to escaping! Maybe because my dad didn't come to my baseball games growing up :)